magento/magento2
# Create csp_whitelist for new relic module

### Description (*)
Fixes CSP issue with new relic module when enabled #34707


### Fixed Issues (if relevant)
1. Fixes magento/magento2#34707

### Manual testing scenarios (*)
1. Enable new relic php module
2. enable new relic module & configure
3. enter to the website
4. CSP warnings appears on the console.

### Contribution checklist (*)
